addons/web/static/src/js/framework/utils.js

```diff
@@ -418,6 +418,14 @@ function swap(array, elem1, elem2) {
     array[i1] = elem2;
 }
 
+function is_email(value, allow_mailto) {
+    // http://stackoverflow.com/questions/46155/validate-email-address-in-javascript
+    var re = /^(([^<>()\[\]\.,;:\s@\"]+(\.[^<>()\[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$/i;
```

Comment:
var re;
if(){re =..}
else{re = ..}
to avoid setting re 2 times in case we enter the if condition